src/pages/[platform]/build-a-backend/storage/download/index.mdx

```diff
@@ -129,12 +129,35 @@ await getUrl({ key: 'filename.txt', options: { expiresIn: 60 } });
 
 Download a file to in-memory buffer.
 
+<BlockSwitcher>
+<Block name="Path">
```

Comment:
nit: This block name could be more descriptive, e.g. "Use path parameter" vs "Use key parameter", wdyt?

Reply:
So disccussed on this with Harshitta. We are thinking to add something like Path (Version 6.x.x) and Key (< Version 6.x.x) whatever version we release this on.  so didnt wanna make it like that. 

But i think we can maybe say `Path(Version 6.x.x & above)` and `Key(To be deprecated)`

@hdworld11 thoughts?

Reply:
I agree that the title could be more descriptive. Maybe we can do "with Path parameter" and "with Key parameter (<= vX)". The version for the Key parameter implies that it's deprecated in versions above it.